docker/README.md

```diff
@@ -0,0 +1,26 @@
+# dellemc-openmanage-ansible-modules
```

Comment:
This file talks about how to pull and run the omam docker image. Isn't it should be added to https://hub.docker.com/r/dellemc/openmanage-ansible-modules ? Why do yo need this README instruction on this GitHub repo where your image is not present?

Reply:
It's a good practice to maintain README instruction along with the Dockerfile on github. Any changes  on this gets reflected on docker repository.